CHANGES.md

```diff
@@ -1,3 +1,27 @@
+# v0.2.0
+
+- Add subcommand `aut protocol contract-abi-path` to print the path of the bundled ABI for the Autonity contract
+- Add subcommand `aut protocol contract-src` to print the URL to the source of the Autonity contract used to generate the bundled ABI
+- Add subcommand `aut protocol contract-version` to print the git commit of the Autonity contract used to generate the bundled ABI
+- Add subcommand `aut protocol contract-address` to print the address of the Autonity contract 
+- Add support for python 3.11 (gh-107)
+- Improve messaging when querying validator info for an address that is not in the validator committee (gh-113)
+- Add support for complex types in contract calls (gh-120)
+
+### Docs
+
+- Add examples how to execute contract calls 
+- Update the development explanation for contributors
+
+### Chore
+
+- Use hatch as python tool to manage builds  (gh-116)
```

Comment:
Annoying that Github [won't render](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests) issue link formatting this way within Markdown files, only within actual issues and discussions... Perhaps it will work in release notes, which is good enough.

```
Note: Autolinked references are not created in wikis or files in a repository.
```

Reply:
replaced with links